app/Resources/translations/default/messages.xlf

```diff
@@ -2960,4 +2960,18 @@ Comment: $this->list_modules_categories['search_filter']['name'] = $this->l('Sea
       </trans-unit>
     </body>
   </file>
+  <file original="modules/ps_emailsubscription/ps_emailsubscription.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="2df860f82be52de57fc5e3f7ecac9431">
+        <source>Newsletter subscription: Unable to delete customer using email.</source>
+        <target>Newsletter subscription: Unable to delete customer using email.</target>
+        <note>Line: 1267</note>
+      </trans-unit>
+      <trans-unit id="0b6664aa3bb3e3130368b120a8adfa91">
+        <source>Newsletter subscription: Unable to export customer using email.</source>
+        <target>Newsletter subscription: Unable to export customer using email.</target>
```

Comment:
Same for this wording.

Reply:
Same here but for the `hookActionExportGDPRData` hook. It's weird you didn't find them do you think I don't have the right version of the module?

Reply:
Seems to be version `2.3.0` and I found the new keys here on `master` branch https://github.com/PrestaShop/ps_emailsubscription/blob/2e3b67bd260d7984403c98fe084bf091134f550b/ps_emailsubscription.php#L1270

Reply:
FYI, both wordings can be found in the 1.7.4 catalog on Crowdin. But not in the following catalogs.

Reply:
So they disappeared from the 1.7.5 version?

Reply:
Yep, it makes sense if the wording cannot be found in ps_emailsubscription.

Reply:
@jolelievre Looks like you are working with outdated sources 🤔 : https://github.com/PrestaShop/ps_emailsubscription/blob/master/ps_emailsubscription.php#L1277

Anyway, let's just remove this wordings from the file and get over with it.